README.md

```diff
@@ -11,7 +11,7 @@
 
 You can use it to:
 * Build a server that can talk in multiple protocols (**on same port**), or access all sorts of services
-  * restful http/https, [h2](https://http2.github.io/http2-spec)/[gRPC](https://grpc.io). using http/h2 in brpc is much more friendly than [libcurl](https://curl.haxx.se/libcurl/). Access protobuf-based protocols with HTTP/h2+json, probably from another language.
+  * restful http/https, [h2](https://httpwg.org/specs/rfc9113.html)/[gRPC](https://grpc.io). using http/h2 in brpc is much more friendly than [libcurl](https://curl.haxx.se/libcurl/). Access protobuf-based protocols with HTTP/h2+json, probably from another language.
```

Comment:
the original rfc referred should be rfc7540, which has been obsoleted by rfc9113. haven't check the changes.

Reply:
What should I do next? I don't see what you mean.

Reply:
httpwg.org/specs/rfc9113.html  come from https://http2.github.io

Reply:
the latest rfc should be ok.